qmail-start.c

```diff
@@ -103,6 +103,7 @@ int main(int argc, char **argv)
   switch(fork()) {
     case -1: die();
     case 0:
+      if (prot_gids(auto_userr, auto_gidq) == -1) die();
```

Comment:
Do we want to do the equivalent for other spawned programs as well?

Reply:
It is already there: the prot_gid() call right after the auto_gid() calls.